src/HotChocolate/AspNetCore/src/AspNetCore/GraphQLServerOptions.cs

```diff
@@ -26,11 +26,21 @@ public sealed class GraphQLServerOptions
     /// </summary>
     public bool EnableGetRequests { get; set; } = true;
 
+    /// <summary>
+    /// Defines if GraphQL HTTP GET requests are allowed.
+    /// </summary>
+    public bool EnforceGetRequestsPreflightHeader { get; set; } = false;
```

Comment:
No default true?

Reply:
So, GET requests are in any case secured. As we only allow queries. But we also people to go for strictness if they want to.